pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -81,6 +81,7 @@ func GetBootstrapSecurityContextConstraints(sccNameToAdditionalGroups map[string
 			SupplementalGroups: kapi.SupplementalGroupsStrategyOptions{
 				Type: kapi.SupplementalGroupsStrategyRunAsAny,
 			},
+			SeccompProfiles: []string{"*"},
```

Comment:
This is the privileged SCC.  I left the other SCCs as is for 3.3 which means no seccomp value can be specified and the kubelet default will be used. 

The discussion upstream is to change the kubelet default to docker/default but since that is untested I was thinking that the better option is to leave this up to the administrator.